lib/contentTypeParser.js

```diff
@@ -297,11 +318,33 @@ function hasContentTypeParser (contentType) {
   return this[kContentTypeParser].hasParser(contentType)
 }
 
+function removeContentTypeParser (contentType) {
+  if (this[kState].started) {
+    throw new Error('Cannot call "removeContentTypeParser" when fastify instance is already started!')
+  }
+
+  if (Array.isArray(contentType)) {
+    contentType.forEach((type) => this[kContentTypeParser].remove(type))
```

Comment:
Can you please use a for..of loop?

Reply:
I will change it later! I used `forEach` because it is also used [here](https://github.com/fastify/fastify/blob/aa58990205ec6923358c5f86a8b0bf55a3f378f0/lib/contentTypeParser.js#L288).